rotkehlchen/tests/unit/test_ethereum_manager.py

```diff
@@ -227,3 +227,11 @@ def test_nodes_sets():
 )
 def test_nodes_speed():
     """TODO"""
+
+
+def test_get_blocknumber_by_time(ethereum_manager):
+    """
+    Queries the graph for known block times
```

Comment:
Can you also do one with `etherscan=True` here to make sure all the bases are covered?